src/hash/sha1/mbedtls.h

```diff
@@ -5,20 +5,15 @@
  * a Linking Exception. For full terms see the included COPYING file.
  */
 
-#ifndef INCLUDE_hash_mbedtld_h__
-#define INCLUDE_hash_mbedtld_h__
+#ifndef INCLUDE_hash_sha1_mbedtld_h__
```

Comment:
`mbedtld` 🙄.

Reply:
Oops, yes

Reply:
Heh, in fact I can point at you here :P

Reply:
(the eyesballs were my own blame-acceptance 😉 )

Reply:
I cannot see any smileys, thus I didn't get that :P